exampleservice/CustomerService/Events/GenericErrorEvent.cs

```diff
@@ -0,0 +1,20 @@
+﻿using exampleservice.Framework.BaseFramework;
+
+namespace exampleservice.CustomerService.Events
+{
+    public class GenericErrorEvent : EventBase
+    {
+        public GenericErrorEvent()
+        {
+            base.Message = "An error occured...";
```

Comment:
Is there any similar code in the blueprint? Adding localized strings to data in events looks like a bad idea to me.

Reply:
the intention was just to use possitive and negative errors - i agree in using localized string, so we should delete it

Reply:
not used anymore. Should we remove it?

Reply:
removed with [commit](https://github.com/mriedmann/simplescript/pull/3/commits/a0f2b646b04a736b377db8dd385be4807dbac41c)